kqueen_ui/utils/fields.py

```diff
@@ -41,6 +43,8 @@ class FileField(SelectableMixin, WTFileField):
 
 
 class IntegerField(SelectableMixin, WTIntegerField):
+    def __init__(self, *args, **kwargs):
+        super(IntegerField, self).__init__(*args, **kwargs)
```

Comment:
This will happen either way, without explicitly calling it like this

Reply:
Done